docs/Concepts/Privacy/Plugin-Privacy.md

```diff
@@ -2,63 +2,75 @@
-   in the same way as a public Ethereum transaction.
+When submitting a private transaction using [`eea_sendRawTransaction`](../../Reference/API-Methods.md#eea_sendrawtransaction),
+the signed transaction must be sent to `0x000000000000000000000000000000000000007a` to indicate the
+[privacy precompiled contract](../Privacy/Private-Transaction-Processing.md) being used.
 
-### Mining Transactions
-When it comes to mining transactions the process happens but in reverse. 
+The transaction flow is as follows:
 
+1. The JSON-RPC endpoint passes the private transaction to the private transaction handler (for example Tessera).
+2. The private transaction handler sends the private transaction to the privacy plugin.
+3. The plugin decides what data to store onchain in the payload, for example encrypt and serialize the private
+    transaction.
+4. The plugin returns what needs to be stored in the payload for the PMT.
+5. The private transaction handler creates a PMT for the private transaction, and propagates the PMT using devP2P in
+    the same way as a public Ethereum transaction.
 
-1. The Mainnet Transaction Processor processes the Privacy Marker Transaction in the same way as
-   any other public transaction. On nodes containing the [privacy precompile contract](../../Reference/API-Methods.md#priv_getprivacyprecompileaddress)
-   specified in the `to` attribute of the Privacy Marker Transaction, the Mainnet Transaction Processor passes the
-   Privacy Marker Transaction to the privacy precompile contract.
+### Mining transactions
+
+When it comes to mining transactions the process happens in reverse to sending transactions.
+
+1. The Mainnet transaction processor processes the PMT in the same way as
+    any other public transaction. On nodes containing the [privacy precompile contract](../../Reference/API-Methods.md#priv_getprivacyprecompileaddress)
+    specified in the `to` attribute of the PMT, the Mainnet transaction processor passes the PMT to the privacy precompile contract.
 
    !!! note
-   Nodes receiving the Privacy Marker Transaction that do not contain the privacy precompile
-   contract specified in the Privacy Marker Transaction ignore the Privacy Marker Transaction.
 
-1. The privacy precompile contract queries your plugin for the private transaction using the privacy marker transaction.
-1. The privacy precompile contract passes the private transaction to the Private Transaction
-   Processor. The privacy group ID specifies the private world state to use.
-1. The Private Transaction Processor executes the transaction. The Private Transaction Processor
-   can read and write to the private world state, and read from the public world state.
+       Nodes receiving the PMT that do not contain the specified privacy precompile
+       contract will ignore the PMT.
+
+1. The privacy precompile contract queries the plugin for the private transaction using the PMT.
```

Comment:
This is a mouthful, but can't think of a better way. 

The precompile passes the pmt to the plugin and wants the private transaction back for that pmt.